lib/core/bootstrap.class.ts

```diff
@@ -14,21 +14,47 @@ export function bootstrap(...BootstrapedApps: Array<IAppStatic>): Array<IApp> {
     const app: IApp = new BootstrapedApp(server)
 
     server.connection(_.extend(Reflect.getMetadata('hapiour:config', BootstrapedApp), DEFAULT_APP_CONFIG))
-    try {app.onInit()} catch(err) {}
+    try {
+      app.onInit()
+    } catch (err) {
+      _.noop()
+    }
     const plugins: Array<IPlugin> = getPluginsRecurs(Reflect.getMetadata('hapiour:plugins', BootstrapedApp))
     const routes: Array<RouteConfiguration> = getRoutesWithConfigRecurs(BootstrapedApp)
     if (_.isEmpty(plugins)) {
-      try {app.onRegister()} catch(err) {}
+      try {
```

Comment:
Nope, _.attempt() require a function, it will only act like a gracefull try/catch.

```
let test = {}
_.attempt(test.myFunc)
// TypeError: Cannot read property 'call' of undefined
```